src/table/selections-utils.js

```diff
@@ -55,7 +58,18 @@ export const getSelectedRows = (selectedRows, qElemNumber, rowIdx, evt) => {
   return selectedRows;
 };
 
-export function selectCell(cell, selState, selDispatch, evt) {
+const removeTabSequence = (rowElements, nextCellCoord) => {
+  const cell = rowElements[nextCellCoord[0]]?.getElementsByClassName('sn-table-cell')[nextCellCoord[1]];
+  cell.setAttribute('tabIndex', '-1');
+};
+
+export function selectCell(cell, selState, selDispatch, focusedCell, setFocusedCell, rootElement, pageInfo, page, evt) {
```

Comment:
I think codeclimate has an valid point here. 9 arguments for a function usually indicates that it is doing too much. We want to keep the focus handling separate from the selection handling

Reply:
Right. Agree. Gonna fix.